trustmanager/keyfilestore.go

```diff
@@ -88,8 +215,15 @@ func (s *KeyFileStore) ExportKey(name string) ([]byte, error) {
 
 // ImportKey imports the private key in the encrypted bytes into the keystore
 // with the given key ID and alias.
+// This is only used for root, so no need to touch the keyInfoMap
```

Comment:
Is this comment true after the #536 was merged?  It looks like the `keyInfoMap` is being set regardless of alias.